packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2038,9 +2080,28 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             summaryLogger,
         );
 
+        /**
+         * Since the latest summary we are tracking is based off of the snapshot we just fetched, update the summary
+         * GC version to the GC version in this snapshot too.
+         */
+        if (this.gcEnabled) {
```

Comment:
@arinwt I am not 100% sure if we need to update summaryGCVersion here. Is it possible that when we force a refresh while summarizing, the GC version of the downloaded snapshot is different from the `currentGCVersion`? From what I understand, it is possible but making sure.

Reply:
Yeah it is possible.